services/wallet/service.go

```diff
@@ -271,6 +267,12 @@ func buildPathProcessors(
 	ensRelease := pathprocessor.NewENSReleaseProcessor(rpcClient, transactor, ens)
 	ret = append(ret, ensRelease)
 
+	ensPublicKey := pathprocessor.NewENSPublicKeyProcessor(rpcClient, transactor, ens)
```

Comment:
@saledjenic I'm guessing this got lost during some rebase, re-adding it here

Reply:
Have no idea, how that was missed. Good that you brought it back.